client/my-sites/plan-features/scroller.jsx

```diff
@@ -0,0 +1,334 @@
+/**
+ * External dependencies
+ */
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import Gridicon from 'gridicons';
+import { clamp, inRange, range, round } from 'lodash';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+
+const MIN_CELL_WIDTH = 240; // px
+const SIDE_PANE_RATIO = 0.12; // 12% of full width
+const MIN_PLAN_OPACITY = 0.4;
+const NO_SCROLL_PADDING = 20; // will appear when plans show up without scrolling
+
+export default class PlanFeaturesScroller extends PureComponent {
```

Comment:
Is this component tightly coupled to plans? Could it be rather a generic component, usable elsewhere? 

It looks like it could just be a matter of renaming a few things, or maybe I'm not looking hard enough.

Reply:
This component doesn't fit general purpose as it depends on the specific DOM structure.